test/Swashbuckle.AspNetCore.SwaggerGen.Test/SwaggerGenerator/SwaggerGeneratorTests.cs

```diff
@@ -50,6 +50,10 @@ public void GetSwagger_GeneratesSwaggerDocument_ForApiDescriptionsWithMatchingGr
                 }
             );
 
+            var provider = subject as ISwaggerDocumentInformation;
+            var documentNames = provider.GetDocumentNames();
+            Assert.Equal(new[] { "v1", "v2" }, documentNames);
```

Comment:
I think this can be changed to something like this:

```csharp
var provider = Assert.IsOfType<ISwaggerDocumentInformation>(subject);
var documentNames = provider.GetDocumentNames();
Assert.Equal([v1", "v2"], documentNames);
```

Reply:
I'm sure you meant Assert.IsType<...>(subject)
but xunit analyzer complained that we should use IsAssignableFrom<T> so, I updated to that instead.